CONTRIBUTING.md

```diff
@@ -10,17 +10,12 @@ A git submodule is essentially a project in another project.
 
 ## Terminology
 
-* _Language_ - the name of a programming language, e.g. _C++_.
-* _Track_ - a collection of exercises in a programming language.
-* _Track ID_ - a url-friendly version of the language name, e.g. `cpp`.
-* _Problem_ - a high-level, language-independent description of a problem to solve.
-* _Implementation_ - a language-specific implementation of a problem. This contains at
-  minimum a README and a test suite.
+Check out the [glossary][] for an overview of the important terms for Exercism, many
+of which are used as class names in this codebase.
```

Comment:
Linking to the glossary is good.

I liked that the trackler specific ones were listed separately here, but maybe a canonical source of information is better even if it is external to the repository.